src/pkey/clu_pkey.c

```diff
@@ -0,0 +1,257 @@
+/* clu_pkey_setup.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL. (formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+#include <wolfclu/clu_header_main.h>
+#include <wolfclu/clu_optargs.h>
+#include <wolfclu/pkey/clu_pkey.h>
+#include <wolfclu/x509/clu_cert.h>
+#include <wolfclu/x509/clu_parse.h>
+
+static void wolfCLU_pKeyHelp(void)
+{
+    printf("./wolfssl pkey\n");
+    printf("\t-in file input for key to read\n");
+    printf("\t-pubout output the public key\n");
+}
+
+
+/* print out PEM public key
+ * returns 0 on success
+ */
+static int wolfCLU_pKeyPEMtoPubKey(WOLFSSL_BIO* bio, WOLFSSL_EVP_PKEY* pkey)
```

Comment:
Future improvement: `wolfCLU_pkeyPemToPubKey`. Stick to camel case after the `wolfCLU_` namespacing prefix.